llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -1360,8 +1360,15 @@ void TargetLibraryInfoImpl::addVectorizableFunctionsFromVecLib(
     addVectorizableFunctions(VecFuncs_DarwinLibSystemM);
     break;
   }
-  case LIBMVEC_X86: {
-    addVectorizableFunctions(VecFuncs_LIBMVEC_X86);
+  case LIBMVEC: {
+    switch (TargetTriple.getArch()) {
+    default:
+      break;
+    case llvm::Triple::x86:
+    case llvm::Triple::x86_64:
+      addVectorizableFunctions(VecFuncs_LIBMVEC_X86);
+      break;
+    }
```

Comment:
```suggestion
    if (TargetTriple.isX86())
      addVectorizableFunctions(VecFuncs_LIBMVEC_X86);
```

Reply:
This PR is prep work for adding support for the AArch64 build of libmvec. We'll want this layout then anyway so I'm going to reject this suggestion.